### DISCUSSION TITLE
Custom NER for new entities - Model differences and NER details

### DISCUSSION THREAD

**Discussion by teresa-arima** [explosion/spaCy#12321] [Help: Best practices]
I am interested in using a Custom NER to differentiate two types of Person names, and I am not interested in the entities the trained NER has, only in the new ones I will train it for. I am considering a model with the tok2ver and NER components. I have some probably quite naÏve questions I cannot find the answer for:
1. What are the differences between employing a spacy blank “es” model or the large model, (es_core_news_lg in my case)?
2. Would there be any differences using the transformer model apart from the obvious performance gains?
3. When I train a custom NER, am I training the model from scratch with only the training set I provide or am I building upon the model’s knowledge? If so, what information learnt from the original corpus would be preserved?
4. In this line, should I also train the tok2ver component?

Thanks!

**Comment by svlandeg** [comment]
Hi @teresa-arima. If you're not interested in any of the NER labels from a pretrained model like the `ner` model from `es_core_news_lg`, then I would advice starting from a new `ner` model all together. But as with most things in ML, the proof is often in the pudding: you could try both approaches and see what works best for your use-case in terms of accuracy/F-score on an independent dev set.

If you want to avoid training from scratch and would like to use some pretrained `tok2vec` layer, you can [source](https://spacy.io/usage/processing-pipelines#sourced-components) it from an existing pipeline and then use a [`Tok2VecListener`](https://spacy.io/api/architectures#Tok2VecListener) within your (new) `ner` component so that it will have pretrained embeddings to start from. You can also have a look at our [`pretrain`](https://spacy.io/api/cli#pretrain) functionality.

If you're relatively new at this all, I advice reading through the [training documentation](https://spacy.io/usage/training) in detail first, and in particular get more familiar with the [`config`](https://spacy.io/usage/training#config) system which allows you to flexibly define and experiment with various different setups for pipelines.
